internal/framework/flex/auto_flatten.go

```diff
@@ -92,6 +92,7 @@ func (flattener autoFlattener) convert(ctx context.Context, vFrom, vTo reflect.V
 
 	case reflect.Interface:
```

Comment:
Is there an `auto_expand` equivalent to this case?

Reply:
@jar-b not yet. `Expand` needs a package specific function to work correctly. Still thinking through how to make that work as easily as possible

Reply:
Makes sense. This is not a blocker (should've indicated as much in the original comment), appreciate the context!